apps/condo/domains/notification/schema/DisconnectUserFromDeviceService.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Generated by `createservice notification.DisconnectUserFromDeviceService --type mutations`
+ */
+const { GQLCustomSchema } = require('@core/keystone/schema')
+
+const access = require('@condo/domains/notification/access/DisconnectUserFromDeviceService')
+const { Device: DeviceAPI } = require('@condo/domains/notification/utils/serverSchema')
+
+const DisconnectUserFromDeviceService = new GQLCustomSchema('DisconnectUserFromDeviceService', {
+    types: [
+        {
+            access: true,
+            type: 'input DisconnectUserFromDeviceInput { dv: Int!, sender: SenderFieldInput!, deviceId: String!, pushTransport: PushTransportType! }',
+        },
+        {
+            access: true,
+            type: 'type DisconnectUserFromDeviceOutput { status: String! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canDisconnectUserFromDevice,
+            schema: 'disconnectUserFromDevice(data: DisconnectUserFromDeviceInput!): DisconnectUserFromDeviceOutput',
+            resolver: async (parent, args, context) => {
+                const { data: { dv, sender, deviceId, pushTransport } } = args
+                const attrs = { dv, sender, owner: { disconnectAll: true } }
+                const existingItem = await DeviceAPI.getOne(context, { deviceId, pushTransport })
```

Comment:
What should happen when `existingItem` is not found?

Reply:
as @pahaz said, nothing will happen actually, have a look at lines 30-32 please.